config.go

```diff
@@ -35,11 +39,33 @@ func defaultSocketPath() string {
 	return filepath.Join(home, ".ssh", "wsl2-ssh-agent.sock")
 }
 
+func where() string {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return filepath.Join(home, ".ssh", "wsl2-ssh-agent.sock")
+}
+
+func commandPath(command string) string {
+	path := os.Getenv("PATH")
+	defer os.Setenv("PATH", path)
+	os.Setenv("PATH", path + ":/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Windows/System32/OpenSSH")
+	path, err := exec.LookPath(command)
+	if err != nil {
+		return ""
+	}
+	return path
+}
+
 func newConfig() *config {
 	c := &config{}
 
 	flag.StringVar(&c.socketPath, "socket", defaultSocketPath(), "a path of UNIX domain socket to listen")
+	flag.StringVar(&c.sshExePath, "ssh-exe", commandPath("ssh.exe"), "a path of Windows ssh.exe")
+	flag.StringVar(&c.powershellPath, "powershell", commandPath("powershell.exe"), "a path of Windows PowerShell (powershell.exe)")
```

Comment:
Umm, I like `--ssh-exe-path` and `--powershell-exe-path`, or simply  `--ssh-path` and `--powershell-path`.